addons/mission/functions/fnc_groundFog.sqf

```diff
@@ -0,0 +1,62 @@
+#include "script_component.hpp"
+/*
+ * Author: TorturedChunk, Kaysi, mmmyum, CNSU, Kresky, Jonpas, Mike
```

Comment:
Who are those people and do we have permission?

Reply:
Remnants from the original Ground Fog Kresky used way back on Halloween.

It's had a lot of edits since then, I know Kresky fiddled with it a bit, you did something and so did I.

Reply:
But do we have permission from those people? Was the function licensed so we can use it?